code/modules/mob/living/carbon/human/combat.dm

```diff
@@ -37,10 +37,7 @@
 	if(src.zone_sel.selecting == "mouth" && !(S.status & ORGAN_DESTROYED) && ishuman(target) && !(T.check_body_part_coverage(MOUTH)) && last_shush + shushcooldown <= world.time)
 		last_shush = world.time
 		T.forcesay("-")
-		target.visible_message( 
-			"<span class='danger'>[src] places a hand over [target]'s mouth!</span>",
-			"<span class='danger'>You place a hand over [target]'s mouth!</span>", 
-			)
+		visible_message("<span class='danger'>[src] places a hand over [target]'s mouth!</span>")
```

Comment:
Shouldn't you have \The [src] and \the [target]?

Reply:
Those macros actually do nothing, since they're inserted by default
We request people use them in most cases due to standards
It looks better
But when `[src]` is usually a proper noun, it's fine

Reply:
Ooh, okay